test/specs/dom/index.js

```diff
@@ -0,0 +1,542 @@
+/** only run DOM tests in browsers */
+if (typeof window !== "undefined") {
+  const PalindromDOM = require("../../../src/palindrom-dom");
+  const assert = require("assert");
+  const moxios = require("moxios");
+  const sinon = require("sinon");
+  const expect = require("chai").expect;
+
+  function createLinkTest(href, parent) {
```

Comment:
I'd rename it to something like `createAndClickOnLink` as this is what we do. `createLinkTest` is not so self-explanatory to me. Same applies for the functions below.